tests/testKinematicsInterval.cpp

```diff
@@ -37,6 +37,7 @@ TEST(Interval, InverseKinematics) {
 
   // Instantiate kinematics algorithms
   auto parameters = boost::make_shared<KinematicsParameters>();
+  parameters->method = OptimizationParameters::Method::UNCONSTRAINED;
```

Comment:
instead of UNCONSTRAINED I would call this SOFT_CONSTRAINTS

Reply:
done